ts/features/itwallet/credentials/store/reducers/index.ts

```diff
@@ -16,6 +26,29 @@ export const itwCredentialsInitialState: ItwCredentialsState = {
   credentials: []
 };
 
+const CURRENT_REDUX_ITW_CREDENTIALS_STORE_VERSION = 0;
+
+export const itwCredentialsStateMigrations: MigrationManifest = {
```

Comment:
Did we lose the migration v1 that adds the `iat` and `exp` claims?

Reply:
I've missed this migration while merging master branch! Addressed here: https://github.com/pagopa/io-app/pull/6225/commits/168ed6c9098fef71931c43f8c7c65efb28ca82c0